articles/translating-jamf-pro-terminology.md

```diff
@@ -19,7 +19,7 @@ Whether you are using Fleet alongside Jamf Pro to manage your Apple devices or p
 | MDM enrolled                   | MDM enabled            | MDM is a feature of Fleet. Hosts can be enrolled in Fleet without MDM enabled. You can get the benefits of Fleet while keeping your devices MDM enrolled with another solution.                                                                                                           |
 | Configuration Profiles         | Custom settings        | Controls > OS settings > Custom settings                                                                                                                                                                                                                                                   |
 | Enrollment customization       | Setup experience       | Controls > Setup experience                                                                                                                                                                                                                                                                |
-| Cloud identity providers       | Single sign-on options | Settings > Single sign-on options                                                                                                                                                                                                                                                          |
+| Cloud identity providers       | Single sign-on (SSO)   | Settings > Single sign-on (SSO)                                                                                                                                                                                                                                                          |
```

Comment:
This change already exists on the `docs-v4.67.0` branch, I have no idea why it's showing up in this diff. Please ignore.